Godeps/_workspace/src/k8s.io/kubernetes/pkg/proxy/hybrid/proxier.go

```diff
@@ -0,0 +1,130 @@
+package hybrid
```

Comment:
Probably shouldn't be upstream for now, unless you're ready to open the PR to Kube.